data/templates/dart.dart

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env scriptisto
+# This compiles "cli/main.dart" from the source directory and uses *this* script to run it.
+#
+# To get started:
+# 1) mkdir -p cli && echo "void main(List<String> arguments) { print('Hello Dart, arguments=' + arguments.toString()); }" >cli/main.dart
```

Comment:
I see what you are doing with this, but can you please clarify why is it better than using "files". Every file there is embedded in the script, so naturally every change counts. 

I would expect a dart template to looks inverse to this: there is a "file" for the shell script. The main body of the script is the dart file. 

Are you trying to introduce the development flow, where scriptisto script is just a "make-and-run.sh", but your actual code is elsewhere?